src/features/content-insights/ContentAnalyticsErrorState.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../styles/variables';
+
+.ContentAnalyticsErrorState {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .ContentAnalyticsErrorState-text {
```

Comment:
minor: do we need to nest this inside of the "ContentAnalyticsErrorState" styling? I vaguely remember something about how this style of class naming (SUIT naming I think it was called) is so that we don't have to nest things too deeply

Reply:
Good catch, I should be able to flatten this